README.md

```diff
@@ -518,11 +568,48 @@ Type: `config => transformedConfig`. Default `x => x` (identify function).
 If you want to make changes to the Webpack configuration beyond what's available in the above options, you can use this, the nuclear option.
 Your function receives the Webpack configuration that Underreact generates and returns a new Webpack configuration, representing your heart's desires.
 
-### vendorModules
+**Tip:** You should think twice before using `webpackConfigTransform`, as Underreact tries its best to abstract away Webpack so that you can focus on the important part of building your application.
 
-Type: `Array<string>`. Default: `[]`.
+## FAQs
 
-Identifiers of npm modules that you want to be added to the vendor bundle.
-The purpose of the vendor bundle is to deliberately group dependencies that change relatively infrequently — so this bundle will stay cached for longer than the others.
+### Why can't I import CSS?
 
-By default, the vendor bundle includes `react` and `react-dom`.
+### How do I deploy my code to a branch like `hey-pages`?
+
+### How do I access siteBasePath in my code?
+
+### Why not use `NODE_ENV`?
+
+Underreact discourages setting of `NODE_ENV` manually, as a number of libraries depend on its value and a wrong value could result in unoptimized builds. You should instead use the cli `mode` option to signal optimization of your bundle. (Internally it would set `NODE_ENV` for your app.)
+
+**If you are used to using `NODE_ENV` to target different deployment environments, you should instead use `DEPLOY_ENV`.**
+
+### How do I use Underreact without React?
+
+### How do I use Mapbox Assembly?
+
+### How do I use a CSS preprocessor like SCSS, LESS?
+
+### How do I upgrade to newer Underreact versions ?
+
+Upgrading to Underreact is as simple as updating a regular node module. The Underreact API should remain relatively stable, but in case of any breaking change we provide you with ways to migrate.
+
+### How do I setup Eslint and Prettier?
+
+### How do I setup testing with Jest?
```

Comment:
Let's focus this question so it's about setting up Jest to use Underreact's Babel configuration.